qualtran/_infra/adjoint.py

```diff
@@ -167,10 +167,6 @@ def build_call_graph(self, ssa: 'SympySymbolAllocator') -> 'BloqCountDictT':
                 counts[bloq.adjoint()] += n
         return counts
 
-    def pretty_name(self) -> str:
-        """The subbloq's pretty_name with a dagger."""
-        return self.subbloq.pretty_name() + '†'
-
     def __str__(self) -> str:
         """Delegate to subbloq's `__str__` method."""
         return f'Adjoint(subbloq={str(self.subbloq)})'
```

Comment:
could we use `str(subbloq) + dagger` in `__str__` instead of `Adjoint(subloq=...)`? It would make the diagrams easier to read.

Reply:
Done.

Reply:
+1, I wanted to do this for a while; but feared the test changes that would need to accompany it